src/test.rs

```diff
@@ -764,10 +764,10 @@ fn split_cert_key() {
         .key_spec(KeySpec::key_exchange())
         .set()
         .unwrap();
-
+    // let mut new_cert = cert;
```

Comment:
You need to clone

let mut new_cert = cert.clone();

At the end new_cert.del_key_container();

Reply:
Fixed.